examples/ui/scrollbars.rs

```diff
@@ -0,0 +1,201 @@
+//! Demonstrations of scrolling and scrollbars.
+
+use bevy::{
+    core_widgets::{ControlOrientation, CoreScrollbar, CoreScrollbarPlugin, CoreScrollbarThumb},
+    ecs::{relationship::RelatedSpawner, spawn::SpawnWith},
+    input_focus::{
+        tab_navigation::{TabGroup, TabNavigationPlugin},
+        InputDispatchPlugin,
+    },
+    picking::hover::Hovered,
+    prelude::*,
+};
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins,
+            CoreScrollbarPlugin,
+            InputDispatchPlugin,
+            TabNavigationPlugin,
+        ))
```

Comment:
So we'll notice if `UiScale` breaks anything again:
```suggestion
        ))
        .insert_resource(UiScale(1.25))
```